app-crypt/tpm2-tss-engine/tpm2-tss-engine-1.1.0_rc0.ebuild

```diff
@@ -3,27 +3,25 @@
 
 EAPI=7
 
-inherit autotools
+inherit autotools bash-completion-r1
 
 DESCRIPTION="OpenSSL Engine for TPM2 devices"
 HOMEPAGE="https://github.com/tpm2-software/tpm2-tools"
-SRC_URI="https://github.com/tpm2-software/${PN}/releases/download/v${PV}/${P}.tar.gz"
+SRC_URI="https://github.com/tpm2-software/${PN}/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz"
```

Comment:
You could put ```${PV/_/-}``` into ```MY_PV```, so you don't have to change it twice in later releases, since you are also using that in ```S```. Otherwise lgtm.

Reply:
Ok I did so